subprojects/docs/src/docs/userguide/jvm/test_report_aggregation_plugin.adoc

```diff
@@ -15,16 +15,95 @@
 [[test_report_aggregation_plugin]]
 = The Test Report Aggregation Plugin
 
-The Test Report Aggregation plugin (plugin id: `test-report-aggregation`) provides tasks and configurations which can be used to aggregate the results of multiple link:{groovyDslPath}/org.gradle.api.tasks.testing.Test.html[Test] task invocations (potentially spanning multiple Gradle projects) into a single HTML report.
-
-The plugin adds configurations and artifact views which collect variants exposed by the JVM Test Suite plugin. More details TBD
+The Test Report Aggregation plugin (plugin id: `test-report-aggregation`) provides tasks and configurations used to aggregate the results of multiple link:{groovyDslPath}/org.gradle.api.tasks.testing.Test.html[Test] task invocations (potentially spanning multiple Gradle projects) into a single HTML report.
 
 [[sec:test_report_aggregation_usage]]
 == Usage
 
-There are two ways to collect test results across multiple subprojects:
+To use the Test Report Aggregation plugin, include the following in your build script:
+
+[.multi-language-text.lang-groovy]
+----
+plugins {
+    id 'test-report-aggregation'
+}
+----
+[.multi-language-text.lang-kotlin]
+----
+plugins {
+    id("test-report-aggregation")
+}
+----
+
+Note that this plugin takes no action unless applied in concert with the <<jvm_test_suite_plugin#jvm_test_suite_plugin,JVM Test Suite Plugin>>.
```

Comment:
Maybe mention that the test suites plugin is automatically applied by the `java` plugin.